### ISSUE TITLE
error: package com.anychart.sample does not exist

### ISSUE THREAD

**Issue by SamsonChoo** [AnyChart/AnyChart-Android#62]
I could import all the packages, but could not import com.anychart.sample.R as it did not find anychart.sample

**Comment by ArsenyMalkov** [comment]
Could you give us more details and describe actions taken step by step?

**Comment by SamsonChoo** [comment]
I was trying to use the thermometer chart (although I noticed the same line of code in other charts), and I copied the sample code. I have included the maven repository and dependency in my build gradle as the instructions have shown. However, I am unable to run the code as it could not "import com.anychart.sample.R", even though I could run the remaining import statements that I copied from the code. 

"error: package com.anychart.sample does not exist" was the error that the studio gave me. In the end I removed the import statement and the corresponding codes and I managed to get my thermometer. I believe it was used for a loading progress bar before the charts show up.
